src/libs/ReportUtils.js

```diff
@@ -571,6 +571,11 @@ function getSmallSizeAvatar(avatarURL, login) {
         return source;
     }
 
+    // Because other urls than CloudFront do not support dynamic image sizing (_SIZE suffix), the given avatar URL is already what we want to use here.
+    if (!CONST.CLOUDFRONT_DOMAIN_REGEX.test(avatarURL)) {
```

Comment:
```suggestion
    if (!CONST.CLOUDFRONT_DOMAIN_REGEX.test(source)) {
```
I think we should test source here.